### PR TITLE
Change AE processor recipes to use SIsg

### PR DESCRIPTION
**Pull request by Dream-Master**
Change Si to SIsg
* https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/pull/11173
* https://github.com/GTNewHorizons/NewHorizonsCoreMod/pull/402

### CODE REVIEW

File: src/main/java/gregtech/loaders/oreprocessing/ProcessingItem.java [hunk 1]
```diff
@@ -22,9 +21,9 @@ public void registerOre(
                 && !aOreDictName.equals("itemNetherQuartz")) {
             switch (aOreDictName) {
                 case "itemSilicon":
-                    GT_OreDictUnificator.addItemData(aStack, new ItemData(Materials.Silicon, 3628800L));
+                    GT_OreDictUnificator.addItemData(aStack, new ItemData(Materials.SiliconSG, 3628800L));
```
Comment (Glease) [inline src/main/java/gregtech/loaders/oreprocessing/ProcessingItem.java:24]: The change of this line is probably not correct. This is to associate an out of GT item (e.g. GalacticCraft Raw Silicon) with oredict data. I don't think they should be Slsg

**Review by Glease — COMMENTED** [review]
I'm in general against a change that would push back AE2 even more, but you should probably already have a good production of these when you have access to AE2, so have quite some mixed feeling about this.

File: src/main/java/gregtech/loaders/oreprocessing/ProcessingItem.java (hunk 1 above)
Comment (Dream-Master) [reply src/main/java/gregtech/loaders/oreprocessing/ProcessingItem.java:24]: yes it not worked. Any idea how we can make it change to Si SG ? @Glease

Comment (Dream-Master) [reply src/main/java/gregtech/loaders/oreprocessing/ProcessingItem.java:24]: Circuits have to use pure silicon not that rwa thing. You can easy make SIsg in mv/hv so it not slow down AE crafting

**Comment by Dream-Master** [comment]
works

**Pull request merged**